nix/.stack.nix/cardano-sl-tools.nix

```diff
@@ -115,7 +115,9 @@
             (hsPkgs.universum)
             (hsPkgs.unordered-containers)
             (hsPkgs.yaml)
-          ] ++ pkgs.lib.optional (!system.isWindows) (hsPkgs.unix);
+          ] ++ (if !system.isWindows
+            then [ (hsPkgs.unix) ]
+            else [ (hsPkgs.Win32) ]);
```

Comment:
Not sure about this one. This was introduced by @angerman few weeks ago directly onto develop. It's a new file that doesn't even exist on `release/2.0.0` (cf: https://github.com/input-output-hk/cardano-sl/tree/release/2.0.0/nix/.stack.nix/cardano-sl-tools.nix)!